.gitignore

```
@@ -27,9 +27,13 @@
 *.lib
 
 # Executables
+*.x
 *.exe
 *.out
 *.app
 
 # Folders and Repositories
 *.vs
+
+# Makefiles
+*Makefile
```

```suggestion
Makefile
```
Unless the `Makefile` has some prefix in the filename, the `*` can be removed.